index.html

```diff
@@ -1,362 +1,657 @@
+                      /></a>
+                    </div>
+                  </div>
+                  <ul class="small-tags">
+                    <li class="canopy-sm-tag">CANOPY</li>
+                    <li>
+                      <img
+                        src="images/Counter.svg"
+                        alt="counter tags"
+                        role="button"
+                        class="counter"
+                      />
+                    </li>
+                    <li class="second-sm-tag">Back End Dev</li>
+                    <li>
+                      <img
+                        src="images/Counter.svg"
+                        alt="counter tags"
+                        role="button"
+                        class="counter"
+                      />
+                    </li>
+                    <li class="second-sm-tag">2015</li>
+                  </ul>
+                  <div class="popup-image-container">
+                    <img src="images/Snapshoot Portfolio.png" alt="pop-image" />
+                  </div>
+                  <div class="popup-mobile-container">
+                   
+                  </div>
+                  <div class="buttom-content-container">
+                    <div class="paragraph-description">
+                      <p class="paragraph-description-desktop">
+                        Lorem Ipsum is simply dummy text of the printing and
+                        typesetting industry. Lorem Ipsum has been the
+                        industry's standard dummy text ever since the 1500s,
+                        when an unknown printer took a galley of type and
+                        scrambled it 1960s with the releaLorem Ipsum is simply
+                        dummy text of the printing and typesetting industry.
+                        Lorem Ipsum has been the industry's standard dummy text
+                        ever since the 1500s, when an unknown printer took a
+                        galley of type and scrambled it 1960s with the releorem
+                        Ipsum is simply dummy text of the printing and
+                        typesetting industry. Lorem Ipsum han printer took a
+                        galley of type and scrambled it 1960s with the releawn
+                        printer took a galley of type and scrambled it 1960s
+                        with the releaLorem Ipsum is simply dummy text of the
+                        printing and typesetting industry. Lorem Ipsum has been
+                        the industry's standard dummy text ever since the 1500s,
+                        when an unknown printer took a galley of type and
+                        scrambled it 1960s with the relea
+                      </p>
+                      <p class="paragraph-description-mobile">
+                        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy
```

Comment:
- [ OPTIONAL ] From the project requirement, you should display all these elements dynamically using javascript :relaxed: so you need to remove all these from your HTML code. I think all this project information should be stored in an array of objects and iterated over displaying them dynamically.